tools/Google.Cloud.Tools.ProjectGenerator/Program.cs

```diff
@@ -359,9 +360,12 @@ private static void GenerateTestProject(ApiMetadata api, string directory, HashS
                     new XElement("PublicSign", new XAttribute("Condition", " '$(OS)' != 'Windows_NT' "), true),
                     new XElement("TreatWarningsAsErrors", true),
                     // 1701, 1702 and 1705 are disabled by default.
-                    // 4014 is required as snippets for streaming samples call Task.Run and don't await the result.
-                    // See https://github.com/googleapis/toolkit/issues/1271 - when that's fixed, we can remove this.
-                    new XElement("NoWarn", "1701;1702;1705;4014")
+                    // xUnit2004 prevents Assert.Equal(true, value) etc, preferring Assert.True and Assert.False, but
+                    //   Assert.Equal is clearer (IMO) for comparing values rather than conditions.
+                    // xUnit2013 prevents simple checks for the number of items in a collection
+                    // AD001 is the error for an analyzer throwing an exception. We can remove this when
```

Comment:
Super-nitty nit: missing a "0" in AD001 in this comment

Reply:
Thanks, will fix.